nanobenchmarks/gen.py

```diff
@@ -38,13 +39,21 @@ def run_experiment(*, blowup: int = 0, parallelism: int = -1, size: int = -1):
     end = time.perf_counter()
     print(f"\n{ret:,}")
     print(f"Time: {end - start:.4f}s")
+    wandb.log({
+        "Execution Time": end - start, 
+        "Input Size": DATA_SIZE, 
+        "Blowup Factor": blowup, 
+        "Parallelism": parallelism, 
+        "Array Size": ret
+        })
     return ret
 
 
 def main():
-    ray.init("auto")
     ray.data.DataContext.get_current().execution_options.verbose_progress = True
 
+    wandb.init(project='gen_spark', entity='ronyw') # change to entity='raysort'
```

Comment:
Run the `black` formatter. Use double quotes.

Reply:
Fixed, thanks Frank!